src/style.css

```diff
@@ -2,0 +1,91 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 20px 8px 0;
+}
+
+.search__bar {
+  box-sizing: border-box;
+  width: calc(100% - 16px);
+  font-family: Avenir, sans-serif;
```

Comment:
Don't fix container size (if there is no such a requirement). Using `calc(100% - 16px)` fixes the width of the `.search__bar` element. If there's no explicit requirement to fix the size, consider using padding or margins on the parent element to control the width indirectly, allowing the content to dictate the size.